src/app/fdctl/config.c

```diff
@@ -142,6 +142,7 @@ static int parse_key_value( config_t *   config,
       }                                                                              \
       config->esection edot ekey[ config->esection edot ekey##_cnt ] = (uint)result; \
       config->esection edot ekey##_cnt++;                                            \
+      return 1;                                                                        \
```

Comment:
???

Reply:
bug fix, it needs to return 1 like all other cases otherwise it errors